src/components/UriViewer.js

```diff
@@ -277,7 +284,7 @@ const UriViewer = _.flow(
         div({ style: { fontSize: 10 } }, ['* Estimated. Download cost may be higher in China or Australia.'])
       ])],
       () => h(Fragment, [
-        isGs(uri) ? 'Loading metadata...' : 'Resolving DOS object...',
+        isGs(uri) ? 'Loading metadata...' : 'Resolving DRS object...',
```

Comment:
Does this need to handle Azure URIs?

Reply:
Removed as part of https://github.com/DataBiosphere/terra-ui/pull/3780#issuecomment-1433494314